docs/src/administration/web/configure-project.md

```diff
@@ -9,10 +9,26 @@ description: |
 
 ## General
 
-From the first page of the project settings, `General`, you can update the project name, or navigate to other project settings options on the left side of the screen.
+From the first page of the project settings, `General`, you can update the project name, set the timezone for the project or navigate to other project settings options on the left side of the screen.
```

Comment:
```suggestion
From the first page of the project settings, `General`, you can update the project name, set the project timezone, and navigate to other project settings options on the left side of the screen.
```

The whole sentence is something of a mess (with `pages` and `options` and all), but I shouldn't get into changing the whole thing yet. :speak_no_evil: